src/Microsoft.Build.Sql/sdk/Sdk.props

```diff
@@ -62,4 +62,16 @@
     <Build Remove="bin/**/*.sql" />
     <Build Remove="obj/**/*.sql" />
   </ItemGroup>
+
+  <ItemGroup>
+      <PackageReference Include="Microsoft.SqlServer.DacFx" Version="161.8089.0" GeneratePathProperty="true" ExcludeAssets="all" />
+      <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" GeneratePathProperty="true" ExcludeAssets="all" />
```

Comment:
Why are you referencing an old, vulnerable version of SqlClient?

Reply:
Just for testing right now